docs/index.md

```diff
@@ -0,0 +1,33 @@
+# Welcome to the documentation for PlantCV-Annotate
+
+## Overview
+
+[PlantCV-annotate](https://github.com/danforthcenter/plantcv-annotate) is composed of modular functions in order to be applicable to a 
+variety of plant types and imaging systems. In the following documentation we will describe use of each function and 
+provide tutorials on how each function is used in the context of an overall image-processing workflow. Development of 
+PlantCV is ongoing---we encourage input from the greater plant phenomics community. Please post questions and comments 
+on the [GitHub issues page](https://github.com/danforthcenter/plantcv-annotate/issues).
+
+## Getting started
+
+The documentation can be navigated using the sidebar table of contents. Documentation for individual PlantCV functions
+are listed under the headings "PlantCV Namespace". For general information on installation,
+updating, and other questions, see:
+
+* [Installing PlantCV](installation.md)
+
+## Contributing 
+
+If you are interested in contributing to PlantCV, please see:
+
+* [Contribution Guide](CONTRIBUTING.md)
+* [Code of Conduct](CODE_OF_CONDUCT.md)
+
+## Versions
+
+The documentation defaults to the `stable` version of PlantCV which is the current release version available from
+PyPI and Bioconda. Documentation for all releases from v1.1 on are also available via the standard Read the Docs 
+popup/pulldown menu (bottom right corner). Select the `latest` version to get the most up-to-date documentation
+associated with the current code in GitHub.
```

Comment:
This is from PlantCV and is out of date. There is no plantcv-annotate on PyPI or Bioconda and there is no stable version nor v1.1 or later